bibliopixel/animation/animation.py

```diff
@@ -2,6 +2,14 @@
 from . runner import Runner
 from .. import log
 from .. threads.animation_threading import AnimationThreading
+from enum import IntEnum
+
+
+class COMPLETE_REASON(IntEnum):
```

Comment:
Ummm: https://github.com/ManiacalLabs/BiblioPixel/blob/master/bibliopixel/drivers/serial/codes.py#L5
It's an enum not a class technically.
I'm OK with that State() plan but we should make it an IntEnum for performance... checking string value is expensive in comparison. You can still print out the value names.